plugins/plugin-annotations/src/scalebox.mjs

```diff
@@ -148,9 +156,10 @@ export const scaleboxMacros = {
       .line(points.__scaleboxMetricTopRight)
       .close()
     // Lead
-    points.__scaleboxLead = points.__scaleboxLead
+    points.__scaleboxLead
       .attr('data-text', so.lead || 'FreeSewing')
       .attr('data-text-class', 'text-sm')
+      .attr('data-text-transform', transform(points.__scaleboxLead))
```

Comment:
Given that font size should be scaling automatically now, are these transforms necessary?

Reply:
They were the last time I tested. I'll try to get this branch in line with `develop` and have a look.

Reply:
Tested the branch, transform is needed.

Reply:
Help me understand? Is this about visual space?

With additional transforms at scale 0.2 and scale 0.5
<img width="915" alt="Screen Shot 2023-05-17 at 4 04 18 PM" src="https://github.com/freesewing/freesewing/assets/5298929/37f4c4d0-3b2f-44c3-a3bb-ce20923e4a52">
<img width="620" alt="Screen Shot 2023-05-17 at 4 05 26 PM" src="https://github.com/freesewing/freesewing/assets/5298929/8ca7d2a8-789e-405f-a6d3-461165cd551a">

Without additional transforms at scale 0.2 and scale 0.5
<img width="1123" alt="Screen Shot 2023-05-17 at 4 04 35 PM" src="https://github.com/freesewing/freesewing/assets/5298929/cca0a069-f5ca-40f7-a23e-a9a4e0ca7f62">
<img width="766" alt="Screen Shot 2023-05-17 at 4 05 05 PM" src="https://github.com/freesewing/freesewing/assets/5298929/cc931888-50fb-4a99-adc2-7983b5e644ab">

Reply:
You're right, and I'm an idiot. Transforms removed.

Reply:
Not an idiot at all. I assume that you fixed the scalebox before fixing the font size and that's how this happened